node/setup/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Setup
+description: 'Installs node, driver dependencies, and builds source'
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 'lts/*'
```

Comment:
This setup action isn't usable once we need something other than LTS, should we make this an input? or do we want to stick to switching away from this when we need other nodes?

Reply:
That seems like something we can do in the future if needed - it's currently only used to set up the repo for our tooling, not for building / testing.